MastodonSDK/Sources/MastodonUI/ViewModel/RelationshipViewModel.swift

```diff
@@ -27,6 +27,7 @@ public enum RelationshipAction: Int, CaseIterable {
     case edit
     case editing
     case updating
+    case showReblogs
```

Comment:
Maybe the `Reblogs` not needs here. Or maybe insert as a low priority relationship. This view model will drive the `ProfileRelationshipActionButton` to display different states.

The priority drive by implicitly enum Int value. So move `showReblogs` before ``

For example, 
If one person is blocking by "myself" and is muting by "myself". Only the high priority state (here is `blocking`) display. Once we unblock the person. The lower priority state display. 

So move the `showReblogs` case before `isMyself` also could be a fix.